### DISCUSSION TITLE
Player Identity > Selection > Pool

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#78] [Framework Discussions]
# Player Identity > Selection > Pool

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
I'm not aware if each team is only able to have 1 specific ship for each player, but in a match there can be players of opposite teams with the same ship.
This game offers specific ships being distributed in tech trees, one for each country, and each player can pick one of these ships (as long as they have the XP and credits/doubloons for it)
